packages/react-canvas-core/src/CanvasEngine.ts

```diff
@@ -19,15 +20,25 @@ export interface CanvasEngineListener extends BaseListener {
 	rendered?(): void;
 }
 
+/**
+ * Defines the CanvasEngine options
+ */
 export interface CanvasEngineOptions {
 	registerDefaultDeleteItemsAction?: boolean;
 	registerDefaultZoomCanvasAction?: boolean;
+	/**
+	 * If set to true debounce the `repaintCanvas` method
+	 */
+	repaintDebounce?: boolean
```

Comment:
I would prefer this to be implicit based on the repaintDebounceMs. As in, if the value is > 0 then we actually apply it without needing the `repaintDebounce ` variable

Reply:
@dylanvorster all right